### PR TITLE
Nerfs bile worms

### PR DESCRIPTION
**Pull request by Jackal-boop**
## About The Pull Request
makes bile worms less shitfuck ass

## Why It's Good For The Game

these little shits wipe out mining teams and spawn in fuck huge pairs in tiny caverns, never de aggro, with full armour pen.

## Changelog

:cl:
balance: bile worms now do 15 damage
/:cl:

### CODE REVIEW

**Comment by Pickle-Coding** [comment]
> these little shits wipe out mining teams and spawn in fuck huge pairs in tiny caverns, never de aggro, with full armour pen.

Can you explain why this is a problem?

**Comment by Jackal-boop** [comment]
> 

they are annoying as hell to fight and not in a challenging or engaging way. they have full armour pen which means your beefy armour you crafted or found is negated. it only takes 3 of their spits to send you back to your moms womb, limping and crying, and they eat you once they kill you. alot of their features by themselves are finem but rolling it all into one enemy that is super fucking common is dumb.

**Comment by san7890** [comment]
spawn in fuck huge pairs in tiny caverns

How does reducing the damage they do help with their spawning pattern?

**Comment by Jacquerel** [comment]
I think this is the issue which needs fixing: https://github.com/tgstation/tgstation/issues/70191 
Bileworms are fine (even quite easy) when they appear on their own, they're only a problem at present because mob spawning in general is a little fucked.
"Fixing" this by adjusting the mob seems a bit silly.

**Comment by Jackal-boop** [comment]
> spawn in fuck huge pairs in tiny caverns
> 
> How does reducing the damage they do help with their spawning pattern?

i dont know how to change spawns and that stuff, reducing their damage will make them manageable, instead of being a near death sentance

File: code/modules/mob/living/basic/lavaland/bileworm/bileworm_actions.dm [hunk 1]
```diff
@@ -73,8 +73,8 @@
 	name = "acidic bile"
 	icon_state = "neurotoxin"
 	hitsound = 'sound/weapons/sear.ogg'
-	damage = 20
-	armour_penetration = 100
+	damage = 15
+	armour_penetration = 50
```
Comment (Fikou) [inline code/modules/mob/living/basic/lavaland/bileworm/bileworm_actions.dm:77]: this in practice does nothing. miners dont have access to 50+ bullet armor

**Comment by ExcessiveUseOfCobblestone** [comment]
can you edit ur title please

**Comment by rasepretep** [comment]
Bileworms primarily wipe out people that run around ignoring fauna, and once they get overwhelmed, they panic instead of just dealing with what they aggro. Even three at a time is easy to deal with provided you keep your cool, the bileworms are likely to kill each other in the crossfire.

But they -do- need some adjusting, specifically the aggro range. I've seen them aggro from as far as 7-8 tiles away, and they never drop the aggro. If you piss off a bileworm and teleport to station, if there is a burrowable tile near where you teleported to, it WILL follow you through time and space.

And as was mentioned, part of the problem with them currently is that Lavaland and Ice Wastes have massively increased spawns of .. everything, due to a bug nobody's been able to locate yet.

**Comment by Shadowflame909** [comment]
full armor pen is dumb. its a boss gimmick. Giving it to a simple fauna that already eats your body is making killing the goliath for armor upgrades pretty pointless

**Comment by IndieanaJones** [comment]
> Both theses end up making it so litterly no body wants to fight theses things while watchers, goliths, and legions all have drops that make them desired to be hunted down and fought past for just one.

An issue that's run into by people making enemies for lavaland is that they really don't want to make anything that would be deemed OP, lest they fuel the fire of the station vs. lavaland argument. This problem is pretty much seen in all the new lavaland monsters (lobstosity, elites, brimdemon, bile worm), I'm not sure what the answer is myself.

**Review by tralezab — CHANGES_REQUESTED** [review]
remove armor pen, or give better loot. lowering damage doesn't seem to be fixing what people are actually talking about, which is negating armor

**Comment by Jacquerel** [comment]
> > Both theses end up making it so litterly no body wants to fight theses things while watchers, goliths, and legions all have drops that make them desired to be hunted down and fought past for just one.
> 
> An issue that's run into by people making enemies for lavaland is that they really don't want to make anything that would be deemed OP, lest they fuel the fire of the station vs. lavaland argument. This problem is pretty much seen in all the new lavaland monsters (lobstosity, elites when their loot isn't the revival stone, brimdemon, bile worm), I'm not sure what the answer is myself.

Honestly I think Watchers do it alright just by virtue of containing "some diamonds",
Just putting some ore inside them would help, it would even make sense that they have ores in their stomach because they're digging through the earth.

**Comment by tralezab** [comment]
> Honestly I think Watchers do it alright just by virtue of containing "some diamonds", Just putting some ore inside them would help, it would even make sense that they have ores in their stomach because they're digging through the earth.

I like it, this would fall under "better loot" if op does it

**Comment by the-orange-cow** [comment]
Idea: random ore type per kill to include rare ones like diamond, bs crystal, bananium(honk).

Also, even with improved loot I think removing the armor pen would be a good idea.

**Comment by rasepretep** [comment]
> Idea: random ore type per kill to include rare ones like diamond, bs crystal, bananium(honk).
> 
> Also, even with improved loot I think removing the armor pen would be a good idea.

I like both of these, with the exception of Bananium. Bileworms are exclusive to Lavaland, which doesn't have bananium ore, and the worms are not found on Icebox. Armor penetration yeah, that should go.

**Comment by Jackal-boop** [comment]
> > Honestly I think Watchers do it alright just by virtue of containing "some diamonds", Just putting some ore inside them would help, it would even make sense that they have ores in their stomach because they're digging through the earth.
> 
> I like it, this would fall under "better loot" if op does it

I like it. What if I make a rare chance for it to drop metal hydrogen? Since it's burrowing deep underground it's gonna be exposed to high pressure something something metal hydrogen

**Comment by tralezab** [comment]
I also have a different plan, to make bileworms "evolve" throughout a round and start weak but after 30 minutes all evolve into "vileworms" which behave as they do right now

**Comment by the-orange-cow** [comment]
> I also have a different plan, to make bileworms "evolve" throughout a round and start weak but after 30 minutes all evolve into "vileworms" which behave as they do right now

If you go that route you should get a new sprite for the evolved form that looks super gross, like the new legions.

**Comment by captainfrost20** [comment]
> remove armor pen, or give better loot. lowering damage doesn't seem to be fixing what people are actually talking about, which is negating armor

Literally this. This plus fixing the spawn rate would make the bileworms less cancer.

**Comment by epicgamer545** [comment]
Finally, we can get a nerf on Bileworms.
Mostly every miner (my source is dchat in previous rounds) that plays nominally wants some sort of change out of this. The issue is that they are WAY too common and can eliminate people in too few shots. I really, really think they should be more rare OR just remove the armor pen (also test it out to make sure everything is fair first). As said before, loot wont solve the problem without removing armor pen.

Also, don't forget the issue that they follow you around without even stopping. You can easily lead them to the mining base and all of the windows will be gone.

**Comment by captainfrost20** [comment]
> Finally, we can get a nerf on Bileworms. Mostly every miner (my source is dchat in previous rounds) that plays nominally wants some sort of change out of this. The issue is that they are WAY too common and can eliminate people in too few shots. I really, really think they should be more rare OR just remove the armor pen (also test it out to make sure everything is fair first). As said before, loot wont solve the problem without removing armor pen.
> 
> Also, don't forget the issue that they follow you around without even stopping. You can easily lead them to the mining base and all of the windows will be gone.

yea the biggest issue is how overwhelming it gets at 3 plus. Literally saw a dude on manuel aggro like 5 of them in a 10 tile area. And there was three more if he went north like 2 more tiles. I aggroed 6 the other day because I didnt see them while running to the base and I only got lucky because they all killed each other.

**Comment by GrimsonGrime** [comment]
Skill issue?

**Comment by JohnFulpWillard** [comment]
> I also have a different plan, to make bileworms "evolve" throughout a round and start weak but after 30 minutes all evolve into "vileworms" which behave as they do right now

you are FUCKED

**Comment by Jackal-boop** [comment]
> I also have a different plan, to make bileworms "evolve" throughout a round and start weak but after 30 minutes all evolve into "vileworms" which behave as they do right now

This seems better then what I could do. I'll close this and let you have at it. If you could implement a metal hydrogen drop in your change that would be cool

**Comment by epicgamer545** [comment]
> I also have a different plan, to make bileworms "evolve" throughout a round and start weak but after 30 minutes all evolve into "vileworms" which behave as they do right now
 
I hope to god we still remove the armor pen

**Pull request closed without merging**